opennms-container/minion/minion-config-schema.yml

```diff
@@ -435,41 +442,41 @@ categories:
               index-type: "field"
               index-field: "parser-name"
             label: "Flow Parser Definition"
-            description: "TODO"
+            description: "Set a parser for flow packets"
             key: "/telemetry/flows/parsers"
-            example: "TODO"
+            example: "Netflow-9-Parser"
         categories:
           # Telemetry :: Flows :: Single Port Listener
           - name: "single-port-listener"
             label: "Single Port Listener"
-            description: "TODO"
+            description: "Configuration for single port flow listener"
             config:
               - name: "enabled"
                 type:
                   name: "boolean"
                 label: "Enable Single Port Listener"
-                description: "TODO"
+                description: "Enable or disable the single port flow listener."
```

Comment:
Note on consistency, most descriptions don't end with a `.` - but there are a few exceptions here and there

Reply:
@bouff thanks, I agree and addressed this topic. Thank you very much!